src/Sources/LightPortal/Env.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bugo\LightPortal;
+
+use Bugo\Compat\Config;
+
+defined('LP_NAME') || define('LP_NAME', 'Light Portal');
+defined('LP_VERSION') || define('LP_VERSION', '2.6.3');
+defined('LP_PLUGIN_LIST') || define('LP_PLUGIN_LIST', 'https://d8d75ea98b25aa12.mokky.dev/addons');
+defined('LP_ADDON_URL') || define('LP_ADDON_URL', Config::$boardurl . '/Sources/LightPortal/Addons');
+defined('LP_ADDON_DIR') || define('LP_ADDON_DIR', __DIR__ . '/Addons');
+defined('LP_CACHE_TIME') || define('LP_CACHE_TIME', (int) (Config::$modSettings['lp_cache_update_interval'] ?? 72000));
+defined('LP_ACTION') || define('LP_ACTION', Config::$modSettings['lp_portal_action'] ?? 'portal');
+defined('LP_PAGE_PARAM') || define('LP_PAGE_PARAM', Config::$modSettings['lp_page_param'] ?? 'page');
+defined('LP_BASE_URL') || define('LP_BASE_URL', Config::$scripturl . '?action=' . LP_ACTION);
```

Comment:
Unfortunately, in this place `$scripturl` is not defined yet

Reply:
Honestly, the intention was not necessarily for it to remain this way. It's just my crazy way of isolating things. 
With a future goal of a standalone script all of the SMF integration needs isolated so as to run only in that context. Just testing what's doin what etc. some times the best way to learn something is to break it.